python/pylibcugraph/setup.py

```diff
@@ -94,46 +62,16 @@ def run(self):
         os.system('rm -rf *.egg-info')
         os.system('find . -name "*.cpp" -type f -delete')
         os.system('find . -name "*.cpython*.so" -type f -delete')
+        os.system('rm -rf _skbuild')
 
 
 cmdclass = dict()
 cmdclass.update(versioneer.get_cmdclass())
 cmdclass["build_ext"] = build_ext
 cmdclass["clean"] = CleanCommand
```

Comment:
Same questions as for cugraph's setup.py. You don't need to set `build_ext`, I'm not 100% sure you need the custom `CleanCommand`, and you should be able to use `versioneer.cmdclass` directly without the `update`.

Reply:
> I'm not 100% sure you need the custom CleanCommand

I think it is useful because whenever I run `setup.py clean`, it doesn't for instance remove `_skbuild` so I added it to the list.

Reply:
Got it, sure that's fine then.